Buildings/Applications/DHC/EnergyTransferStations/CoolingDirectUncontrolled.mo

```diff
@@ -0,0 +1,192 @@
+within Buildings.Applications.DHC.EnergyTransferStations;
+model CoolingDirectUncontrolled
+  "Direct cooling ETS model for district energy systems without in-building 
+  pumping or deltaT control"
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+    redeclare final package Medium1 = Medium,
+    redeclare final package Medium2 = Medium,
+    final m1_flow_nominal = m_flow_nominal,
+    final m2_flow_nominal = m_flow_nominal,
+    show_T = true);
+
+ replaceable package Medium =
+   Modelica.Media.Interfaces.PartialMedium "Medium in the component";
+
+  parameter Modelica.SIunits.MassFlowRate m_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate";
+
+  // pressure drops
+  parameter Modelica.SIunits.PressureDifference dpSup(
+    final min=0,
+    displayUnit="Pa")=50
+  "Pressure drop in the ETS supply side";
```

Comment:
use a higher default value. `50 pa` seems to be a very short section of pipe.

Reply:
It is. The pipe length (in the mechanical room) for a direct connection ETS like this shouldn't be too long, although I agree this default may be too low. I'll adjust after some additional research.